src/pandas_profiling/version.py

```diff
@@ -1,2 +1,2 @@
 """This file is auto-generated by setup.py, please do not alter."""
-__version__ = "3.3.0"
+__version__ = "3.3.1"
```

Comment:
> """This file is auto-generated by setup.py, please do not alter."""

Reply:
I didn't have a recollection of editing this file, so I checked it. The change was done by the CI/CD code formatting step
https://github.com/ydataai/pandas-profiling/pull/1074/commits/91d4d5739e984412ab503c8003bd2b5818222d23

Reply:
this should have been like this in the master, when the release was done, which means that the latest release is wrong 😕 https://github.com/ydataai/pandas-profiling/blob/master/src/pandas_profiling/version.py

it should be 3.3.1

Reply:
never mind, my mistake. why did it change? do you know?

Reply:
it was due to the setup.py have already the 3.3.1 as a version. the automatic update of the version.py was moved to the PR CI/CD since all changes should happen as PR, even version bumps